internal/webhooks/frrconfiguration_webhook.go

```diff
@@ -151,6 +158,21 @@ func validateConfig(frrConfig *v1beta1.FRRConfiguration) error {
 				// shouldn't happen for creates as it would be considered an update, and in any case
 				// we add the updated one at the end because for updates we don't want the old and updated resources
 				// to be considered together.
+				for _, rold := range cfg.Spec.BGP.Routers {
+					for _, rnew := range frrConfig.Spec.BGP.Routers {
+						if rold.ID != rnew.ID {
+							continue
+						}
+						for _, nold := range rold.Neighbors {
+							for _, nnew := range rnew.Neighbors {
+								// router bgp 7003' -c 'neighbor 11.11.11.1 graceful-restart'
```

Comment:
is this a leftover?

Reply:
not really, this was on purpose for discussion. Given the config command has asn, ip and gr those should be the two things I compare. Do you agree? ( I removed the comment)